src/column.rs

```diff
@@ -192,6 +256,39 @@ mod test {
         Ok(())
     }
 
+    #[test]
+    #[cfg(feature = "column_metadata")]
+    fn test_columns_with_metadata() -> Result<()> {
+        use super::Column;
+
+        let db = Connection::open_in_memory()?;
+        let query = db.prepare("SELECT * FROM sqlite_master")?;
+
+        let col_mets = query.columns_with_metadata();
+        for col in &col_mets {
+            assert_eq!(&col.database_name.map(str::to_lowercase), &Some("main".to_owned()));
```

Comment:
Why `to_lowercase` ?

Reply:
The reason for `to_lowercase` here and in the other places is because I believe that when working with sqlite db / table / column names you should be case insensitive, right?